skins/shared/functions.scss

```diff
@@ -11,3 +11,94 @@
 	}
 	@return nth($range, 2);
 }
+
+@function luminance($color) {
```

Comment:
Thanks for providing them. How are you plan to pick alternative if decided to pick one? ;)